### PR TITLE
2788 Code changes for creating base solution & project skeleton

### PR DESCRIPTION
**Pull request by Avani13709**
Code changes for base project skeleton (Exchange set web api project, Functional test project, Unit Test project , Common class library project)
Basic pipeline

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+# MSBuild Binary and Structured Log
+*.binlog
+
+# NVidia Nsight GPU debugger configuration file
+*.nvuser
+
+# MFractors (Xamarin productivity tool) working folder 
+.mfractor/
+/UKHO.FileShareService.API/UKHO.FileShareService.API/Logs
```
[the remark this answers is not in the document]
Comment (Avani13709) [reply .gitignore]: Done and committed

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd).$(BuildCounter)
```
Comment (WorkRD) [inline azure-pipelines.yml:5]: name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd).$(BuildCounter)
```

Comment (WorkRD) [inline azure-pipelines.yml:5]:
```suggestion
name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd).$(BuildCounter)
```

**Review by WorkRD — APPROVED** [review]

**Review by janvi13603 — APPROVED** [review]
Looks good.

**Pull request merged**